### PR TITLE
Wordcloud tokenisation

### LINKED ISSUE #2023: Use ES tokenizer in wordcloud
**Is your feature request related to a problem? Please describe.**
For the purpose of searching and most visualisations, documents are tokenised by Elasticsearch, but the wordcloud uses its own tokeniser (using [CountVectorizer](https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.CountVectorizer.html) ). This is more primitive than the one used by Elasticsearch, and does not work well for all languages.

**Describe the solution you'd like**
Use the termvectors API to get token counts from Elasticsearch.

**Describe alternatives you've considered**
- You can also pass a custom tokeniser function to the CountVectorizer, but then we would need to reverse-engineer the tokeniser we already have.
- You can turn off the wordcloud for some languages (admittedly not a great loss to science). This is what I did as a quick fix in #2012

### PR DESCRIPTION
**Pull request by lukavdplas**
close #2023 

This requests the tokenised text for the wordcloud from Elasticsearch, which is more consistent than using the `CountVectorizer`.

### CODE REVIEW

**Review by Meesch — APPROVED** [review]
Nice work!
